src/routes/demos/save/.Part0DemoFields.svelte

```diff
@@ -8,18 +8,17 @@
 
 <div class="columns is-multiline">
   <div class="column is-full">
-    <h2 class="title">Create a New Demo</h2>
+    <h2 class="title">Create a New Kiosk Instance</h2>
```

Comment:
Instance seems very vague - I would recommend sticking with "Demo", we may need to discuss internally as a team if this needs to be changed